homeassistant/components/history_stats/sensor.py

```diff
@@ -124,35 +124,40 @@ def __init__(
         self._name = name
         self._unit_of_measurement = UNITS[sensor_type]
```

Comment:
Not sure if you want to include setting these to `_attr_name` and `_attr_native_unit_of_measurement` here.

Reply:
That's coming in one of the next parts https://github.com/home-assistant/core/pull/70131

Reply:
https://github.com/home-assistant/core/pull/70255